src/main/java/com/mraof/minestuck/event/ServerEventHandler.java

```diff
@@ -166,6 +177,72 @@ public static void onEntityDamage(LivingHurtEvent event)
 		}
 	}
 	
+	@SubscribeEvent(priority = EventPriority.NORMAL, receiveCanceled = false)
+	public static void onPlayerInjured(LivingHurtEvent event)
+	{
+		if(event.getEntityLiving() instanceof PlayerEntity)
+		{
+			PlayerEntity injuredPlayer = ((PlayerEntity) event.getEntity());
+			Title title = PlayerSavedData.getData((ServerPlayerEntity) injuredPlayer).getTitle();
+			if(title != null)
```

Comment:
Mostly intentional but willing to change, not too important to me

Reply:
If you want to have some sort of difference between a weak/strong effect and no effect, I'd rather suggest it be based around an echeladder rung, instead of the existence or nonexistence of a title.

Reply:
Removing the null check entierly is not great as you still need to check if it's null before checking if it's doom. A simple solution would be to introduce `boolean isDoom = title != null && title.getHeroAspect() == EnumAspect.DOOM` and then replace any other occurences of `title.getHeroAspect() == EnumAspect.DOOM` with `isDoom`

Reply:
It is done